backend/app/controllers/client/practitioners.js

```diff
@@ -1,96 +1,16 @@
 const prisma = require('../prisma-client');
 
-const index = async (ctx) => {
-  const allPractitioners = await prisma.practitioner.findMany();
-  ctx.body = {
-    data: allPractitioners
-  };
-  ctx.status = 200;
-};
-
 const get = async (ctx) => {
   const id = parseInt(ctx.params.id);
   const practitioner = await prisma.practitioner.findUnique({
     where: {
       id: id
     }
   });
```

Comment:
Good catch, that could be a potential security issue

Reply:
I think this probably goes for all of the endpoints that return a Client or Practitioner. No need to expose the auth0sub since every client app already has access to their own if they need it.